src/components/ui-shell/platform-side-nav.hbs

```diff
@@ -6,8 +9,45 @@
   <div class="bx--side-nav__content">
     {{> platform-side-nav-title-bar sidenav=sidenav }}
     <div class="bx--side-nav__scroll">
-      {{> platform-side-nav-nav sidenav=sidenav }}
+      <nav class="bx--side-nav__nav">
+        <ul class="bx--side-nav__nav-items">
+          {{#each sidenav.links}}
+          <li class="bx--side-nav__nav-item">
+            {{#if this.active}}
+            <div class="bx--side-nav__category bx--side-nav__category--active">
+            {{else}}
+            <div class="bx--side-nav__category">
+            {{/if}}
+              <div class="bx--side-nav__category-header">
```

Comment:
Did you have add a level of indent here (toward `#each`) by accident...?

Reply:
🤔 l21 is indented because it's a child of nav__category.

Reply:
Got it, my apologies for my false alarm!. Another discussion made me understand where my finding came from.